content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,140 @@
+ensure the :abbr:`IoT (Internet of Things)` system, and its processes, stay up-to-date.
+
+Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to :menuselection:`IoT app -->
+IoT Boxes`, and clicking on the :guilabel:`IP address` of the :abbr:`IoT (Internet of Things)` box.
+Then, click on :guilabel:`Update` (next to the version number).
+
+If a new version of the :abbr:`IoT (Internet of Things)` box image is available, an
+:guilabel:`Upgrade to _xx.xx_` button appears at the bottom of the page. Click this button to
+upgrade the unit, at which point the :abbr:`IoT (Internet of Things)` box flashes itself to the
+newer version. All of the previous configurations are then saved.
+
+.. note::
+   This process can take more than 30 minutes. Do **not** turn off, or unplug, the :abbr:`IoT
+   (Internet of Things)` box, as it would leave it in an inconsistent state. This means the
+   :abbr:`IoT (Internet of Things)` box needs to be :ref:`re-flashed <iot/config/flash>` with a new
+   image.
+
+   .. seealso::
+      :ref:`iot/config/flash`.
+
+.. image:: updating_iot/flash-upgrade.png
+   :align: center
+   :alt: IoT box software upgrade in the IoT Box Home Page.
+
+Handler (driver) update
+=======================
+
+There may be some instances where drivers or interfaces need to be updated for individual devices
+(e.g. scales, measurement tools, etc.). The IoT handler's (drivers and interfaces) code can be
+modified by syncing them with the configured server handler's code.
+
+This can be helpful in instances where :abbr:`IoT (Internet of Things)` devices (e.g. scales,
+measurement tools, etc.) are not working properly with the :abbr:`IoT (Internet of Things)` box.
+
+For both the Windows :abbr:`IoT (Internet of Things)` (Odoo 16 and higher) and physical :abbr:`IoT
+(Internet of Things)` box, this process can be performed manually from the :abbr:`IoT (Internet of
+Things)` box homepage. Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to
+:menuselection:`IoT app --> IoT Boxes`, and clicking on the :guilabel:`IP address` of the :abbr:`IoT
+(Internet of Things)` box.
+
+Next, click :guilabel:`Handlers list`, and then select :guilabel:`Load Handlers` at the bottom of
+the page.
+
+.. image:: updating_iot/load-handlers.png
+   :align: center
+   :alt: Handlers list on an IoT box with the load handlers button highlighted.
+
+.. important::
+   Handler's code is fetched from the configured server, and it needs to be up-to-date to have the
+   latest fixes and patches.
+
+.. note::
+   A handler update is also performed automatically each time the :abbr:`IoT (Internet of Things)`
+   box is restarted. The only exception to this process is if the :guilabel:`Automatic drivers
+   update` is unchecked in the form view of the :abbr:`IoT (Internet of Things)` box on the Odoo
+   server. This setting can be reached by going to :menuselection:`IoT App --> Select the IoT box
+   --> Automatic drivers update`.
```

Comment:
This guilabel should be italics since the active instruction to navigate to the page where this field is present on the UI comes after you state the field name
```suggestion
   box is restarted. The only exception to this process is if the *Automatic drivers update* is
   unchecked in the form view of the :abbr:`IoT (Internet of Things)` box on the Odoo server. This
   setting can be reached by going to :menuselection:`IoT App --> Select the IoT box --> Automatic
   drivers update`.
```